syntax = "proto3";

message VideoDataProto{
    repeated bytes video_frames = 1; // Repeated fields for storing binary frames (= 1 -> Field 1 belongs to video_frames byte sequence)
    bytes single_frame = 2; //  A single frame as binary
    int32 width = 3;
    int32 height = 4;
    double fps_rate = 5;
    bool frame_size_captured = 6;
}